content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,393 @@
+
+- Admin (root) access to a Linux system (virtual, cloud, or physical instance) running `systemd`.
+- Access to the Internet to download the various components needed for the tutorial.
+- Required CLI tools: `curl`, `tar`.
+- Suggested CLI tools: `jq`.
+
+
+<!-- lessoncontent -->
+
+## Download, install, and configure the components
+
+{{< note >}}
+Download the latest available versions of the required packages.
+The relative path to the downloaded software is assumed to be the `sources` directory.
+Run all the commands as the `root` user.
+{{< /note >}}
+
+```shell
+mkdir sources
+cd sources/
+```
+
+### CNI plugins
+
+Download: [[releases]](https://github.com/containernetworking/plugins/releases)
+
+```shell
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+```
+
+Install:
+
+```shell
+mkdir -p /opt/cni/bin
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+```
+
+Quick test:
+
+```shell
+/opt/cni/bin/host-local --version
+```
+
+The output is similar to:
+
+```
+CNI host-local plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+Configure:
+
+```shell
+mkdir -p /etc/cni
+
+cat << EOF > /etc/cni/cni.template
+{
+  "name": "containerd",
+  "cniVersion": "1.0.0",
+  "plugins": [{
+    "type": "ptp",
+    "ipMasq": true,
+    "ipam": {
+      "type": "host-local",
+      "subnet": "{{.PodCIDR}}",
+      "routes": [
+        {"dst": "0.0.0.0/0"}
+      ]
+    }
+  },{
+    "type": "portmap",
+    "capabilities": {
+      "portMappings": true
+    }
+  }]
+}
+EOF
+```
+
+### containerd
+
+{{% thirdparty-content %}}
+
+Enable IPv4 packet forwarding:
+
+```shell
+cat <<EOF > /etc/sysctl.d/k8s.conf
+net.ipv4.ip_forward = 1
+EOF
+
+sysctl --system
+```
+
+The output is similar to:
+
+```
+...
+* Applying /etc/sysctl.d/k8s.conf ...
+net.ipv4.ip_forward = 1
+* Applying /etc/sysctl.conf ...
+```
+
+Download: [[releases]](https://github.com/containerd/containerd/releases)
+
+```shell
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-amd64.tar.gz
+curl -LO https://raw.githubusercontent.com/containerd/containerd/main/containerd.service 
+```
+
+Configure:
+
+```shell
+mkdir -p /etc/containerd
+
+cat << EOF > /etc/containerd/config.toml
+version = 2
+
+[plugins."io.containerd.grpc.v1.cri".cni]
+
+    # binaries
+    bin_dir = "/opt/cni/bin"
+
+    # configure pod networking
+    conf_template = "/etc/cni/cni.template"
+EOF
+```
+
+Install:
+
+```shell
+tar Cxzvf /usr/local/ containerd-1.7.22-linux-amd64.tar.gz
+mv containerd.service /etc/systemd/system/
+
+systemctl enable containerd.service
+systemctl start containerd.service
+```
+
+Quick test:
+
+```shell
+systemctl status containerd.service
+```
+
+Detailed service check:
+
+```shell
+journalctl -u containerd
+```
+
+### Kubelet
+
+Download: [[latest stable release]](https://cdn.dl.k8s.io/release/stable.txt)
```

Comment:
I'd rather not encourage bookmarking this URL.
```suggestion
Download the latest stable release (to see versions available, visit the [downloads](/releases/download/) page.
```

Reply:
Done